client/src/components/Card/Card.css

```diff
@@ -10,6 +10,7 @@
   /* Hover effect: Scale the card on hover */
   &:hover {
     transform: scale(1.05);
+    transition: transform 0.3s ease-in-out; /* Add a smooth transform transition on hover */
```

Comment:
You can add another effect `.grid-container {
  display: grid;
  grid-template-columns: repeat(
    auto-fill,
    minmax(300px, 1fr)
  ); /* Create a responsive grid with minimum width of 300px */
  gap: 2rem; /* Add a gap between grid items */
  justify-content: center; /* Center the grid items horizontally */
}`

Reply:
Okay